outlines/generator.py

```diff
@@ -91,8 +106,51 @@ def stream(self, prompt, **inference_kwargs):
 def Generator(
     model: Union[SteerableModel, BlackBoxModel],
     output_type: Optional[Any] = None,
+    *,
+    logits_processor: Optional[OutlinesLogitsProcessor] = None,
+    fsm: Optional[interegular.fsm.FSM] = None,
```

Comment:
I would prefer a separate output type instead of a kwarg here.

Reply:
So keep what we had before, right?

Reply:
Yes we could have a `FSM` object like we have `CFG` or `JsonSchema`